shenyu-dist/shenyu-agent-dist/src/main/resources/conf/shenyu-agent.yaml

```diff
@@ -18,13 +18,13 @@
 appName: shenyu-agent
 supports:
   tracing:
-    - jaeger
+#    - jaeger
 #    - opentelemetry
 #     - zipkin
   metrics:
-    - 
+#    - prometheus
   logging:
-    - rocketmq
+#    - rocketmq
```

Comment:
why remove this

Reply:
because users don't configur rocketmq, will report error in log. When users has rocketmq, they can remove annotation